source/beamline/BioXAS/BioXASSide32ElementGeDetector.cpp

```diff
@@ -0,0 +1,30 @@
+#include "BioXASSide32ElementGeDetector.h"
+
+BioXASSide32ElementGeDetector::BioXASSide32ElementGeDetector(const QString &name, const QString &description, AMPVControl *triggerControl, BioXASZebraPulseControl *pulseControl, QObject *parent) :
+	BioXAS32ElementGeDetector(name, description, triggerControl, pulseControl, parent)
+{
+	// Stuff required by AMXRFDetector.
+
+	acquisitionStatusControl_ = new AMReadOnlyPVControl("Status", "DXP1607-I22-01:DetectorState_RBV", this);
+
+	for (int i = 0; i < 20; i++){
```

Comment:
Only the first 20 channels are used on the Side detector. I think this is the way to ensure that the other channels are never used. I'll add a comment, would that make it clearer?